### DISCUSSION TITLE
Defining perspectives and customviews

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#17862] [Q&A / Support] [answered]
To define Perspectives and Custom Views, should we use customviews.php and perspectives.php files or yaml files?

The perspective editor refers to these files: https://github.com/pimcore/perspective-editor/blob/1.x/doc/01_Installation.md#bundle-installation
but there was a PR deleting these files on the demo site: https://github.com/pimcore/demo/pull/380

**Comment by kingjia90** [comment, marked as answer]
it's recommended to use YML as it was migrated to the LocationAwareConfigRepository, any reference of `.php` must be considered outdated/legacy.

https://github.com/pimcore/admin-ui-classic-bundle/blob/9258d42920dbb475badc1adea59a7552ab089ac4/src/CustomView/Config.php#L30-L31

**Reply by cancan101** [reply]
How do I enable adding of new perspectives or views through the UI? Currently that option is disabled:
![image](https://github.com/user-attachments/assets/4c9de90e-4bd5-4eae-9bf8-6717b920a7aa)

If I use:
```yaml
            write_target:
                type: 'settings-store'
```
then how can I migrate my settings from one server to another? There does not appear to be an export option in the editor.
